src/components/Board/SymbolSearch/SymbolSearch.component.js

```diff
@@ -232,14 +232,21 @@ export class SymbolSearch extends PureComponent {
 
   handleSuggestionsClearRequested = () => {};
 
-  handleSuggestionSelected = (event, { suggestion }) => {
+  handleSuggestionSelected = async (event, { suggestion }) => {
```

Comment:
I don't think this is the right place to make the update, as you will be updating all the suggestions (including global symbols). You should make the update on the fetch arasaac suggestions level.